src/meta/src/manager/catalog/mod.rs

```diff
@@ -832,8 +831,37 @@ impl CatalogManager {
             .await;
     }
 
-    /// This is used for both `CREATE TABLE` and `CREATE MATERIALIZED VIEW`.
-    pub async fn start_create_table_procedure(
+    /// This is used for both `CREATE TABLE`
+    pub async fn start_create_table_procedure(&self, table: &Table) -> MetaResult<()> {
```

Comment:
IIUC tables and materialized views notify FE differently because we expect tables to be created quickly while materialized views need to backfill which could be time-consuming.

Reply:
Yes